onnx/common/ir.h

```diff
@@ -311,6 +314,9 @@ struct Value {
   const std::vector<Dimension>& sizes() const {
     return sizes_;
   }
+  std::vector<Dimension>& sizes() {
```

Comment:
This breaks encapsulation. If you need write access, it makes more sense to keep it more strict, for example have a 

    setSize(index, Dimension)

or a

    setSizes(vector)